frontend/src/App.js

```diff
@@ -31,6 +34,9 @@ class App extends Component {
                         <Route exact path="/not-found" component={PageNotFound}/>
                         <Route exact path="/login" component={Login}/>
                         <Route exact path="/register" component={Registration}/>
+                        <Route path="/booking/:serviceproviderid/:serviceid" component={BookingCalendarConsumer}/>
+                        <Route path="/serviceprovider/:serviceproviderid/calendar" component={BookingCalendarServiceProvider}/>
+                        <Route path="/booking/:serviceproviderid" component={Booking}/>
```

Comment:
```suggestion
                        <Route exact path="/booking/:serviceproviderid/:serviceid" component={BookingCalendarConsumer}/>
                        <Route exact path="/serviceprovider/:serviceproviderid/calendar" component={BookingCalendarServiceProvider}/>
                        <Route exact path="/booking/:serviceproviderid" component={Booking}/>
```
Raczej w ten sposób, inaczej przykładowo ścieżka /booking/1/xxx/yyy/zzz będzie działać identycznie jak /booking/1.

Do przemyślenia też na przyszłość postać tych ścieżek, docelowo powinny być raczej elementem hierarchii kategorii dostępnych z navbara. To ustalimy jednak jak będziemy mieć już większą ilość widoków.

Reply:
Jasne, już się raz spotkałem, że przenosiło mnie do innej strony niż chciałem i pozmieniałem po prostu ścieżki :D Teraz przynajmniej wiem w czym rzecz. Exact oczywiście póki co dopisałem, ale pewnie wrzucimy to do tych hierarchii